bin/transync

```diff
@@ -41,13 +41,13 @@ try {
 }
 
 try {
-  var toDictionary = yaml.safeLoad(toFile, { json: true })
+  var toDictionary = toFile === '' ? {} : yaml.safeLoad(toFile, { json: true })
```

Comment:
if the `toFile` doesn't exist, the `toDictionary` becomes undefined and that produced some errors when trying to access `to[key]` later on. This will make `toDictionary` actually a dictionary whether the `toFile` exists or not.